src/main/scala/nl.knaw.dans.easy.deposit/Errors.scala

```diff
@@ -34,6 +34,13 @@ object Errors extends DebugEnhancedLogging {
 
   case class ConfigurationException(msg: String) extends IllegalArgumentException(s"Configuration error: $msg")
 
+  case class LeftoversOfForcedShutdownException(dir: File)
+    extends FileAlreadyExistsException(
+      s"Staging area [$dir] should be empty unless force shutdown during an upload/submit request." +
+        " Directories containing bags are aborted submits, somehow (allow) resubmit if indeed not in fedora." +
```

Comment:
I don't really understand this part of the sentence: `Directories containing bags are aborted submits`